fssimulator: #namespace

    n_actions: 2 # We have 2 Continous actions   Steering, Speed
    n_observations: [480, 640, 3] # We have 6 different observations  screen_size = [480, 640, 3]

    speed_step: 1.0 # Time to wait in the reset phases

    init_linear_forward_speed: 0.0 # Initial linear speed in which we start each episode
    init_linear_turn_speed: 0.0 # Initial angular speed in shich we start each episode


    forwards_reward: 10 # Points Given to go forwards
    turn_reward: 10 # Points Given to turn as action
    end_episode_points: -50 # Points given when ending an episode
    

